src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round((Math.PI * radius * radius) * 10) / 10;
```

Comment:
Using `Math.round()` here may lead to loss of precision for the area calculation. It would be better to format the output when displaying the area rather than rounding it in the calculation method itself. This way, the `calculateArea()` method returns the most accurate result possible.